pkg/controller/init_frontend.go

```diff
@@ -1,21 +1,53 @@
 package controller
 
 import (
-	devtypes "github.com/longhorn/go-iscsi-helper/types"
+	"fmt"
 
+	devtypes "github.com/longhorn/go-iscsi-helper/types"
 	"github.com/longhorn/longhorn-engine/pkg/frontend/rest"
 	"github.com/longhorn/longhorn-engine/pkg/frontend/socket"
 	"github.com/longhorn/longhorn-engine/pkg/frontend/tgt"
 	"github.com/longhorn/longhorn-engine/pkg/types"
 )
 
-var (
-	Frontends = map[string]types.Frontend{}
+const (
+	defaultScsiTimeout       = 60 // SCSI device timeout, unit: seconds
+	defaultIscsiAbortTimeout = 15 // unit: seconds
+
+	defaultEngineReplicaTimeout = 8  // unit: seconds
+	minEngineReplicaTimeout     = 8  // unit: seconds
+	maxEngineReplicaTimeout     = 30 // unit: seconds
```

Comment:
Probably we can just use `Time.Duration`, so we don't need to add these comments, but all of them are self-explanatory.

Reply:
Addressed. Thank you.